### ISSUE TITLE
Feature request for RemBackground

### ISSUE THREAD

**Issue by kakachiex2** [melMass/comfy_mtb#164]
### Describe the problem

Your Rem background node is less precise than others more accurate version like BRIA and BiRefNet
- Cons >Your version in some images create hole inside the image it don't detect the border accurately 
- Pro > I like that it let user change the background color :: useful for 3DTech like Comfy3DPack, TripoSR etc...
- 
![Screenshot 2024-03-24 122115](https://github.com/melMass/comfy_mtb/assets/1762112/3f303a60-5b44-4bad-b6c0-0d25f0f6f988)

### Describe the solution you'd like

Implement others Rem-Background model like in the screenshot so we have more option with the ability to change Back. color
![Screenshot 2024-03-24 121904](https://github.com/melMass/comfy_mtb/assets/1762112/9f3674ad-7b2d-4756-9ad5-aed9e1090f8e)

### Alternatives considered

![Screenshot 2024-03-24 121904](https://github.com/melMass/comfy_mtb/assets/1762112/07ac3078-abef-4d2e-9d85-df0563e80a6b)

**Comment by kakachiex2** [comment]
BiRefNet do a great job in separate the subject from the background no hole, the same from RemBG from LayerStyle node.
By the way thanks a lot for you node pack really useful great job.
![Screenshot 2024-03-24 125032](https://github.com/melMass/comfy_mtb/assets/1762112/3c16604f-6833-4a44-a954-86fcbd42ba0c)

**Comment by melMass** [comment]
Hi,

BiRefNet is planned but I don't think I will implement other methods. 
I did not know about LayerStyle but their implementation looks good and they already bundle multiple variants (rembg, briar etc..)
The node in mtb is a simple implementation of the original [RemBg](https://github.com/danielgatis/rembg) that seem to have since received a few nice updates, I will also fetch those.

TLDR, yes for BiRefNet and updating rembg's update but for other methods I suggest dedicated extensions like LayerStyle and VideoMatting
